wiki/Beatmap/Beatmap_description/es.md

```diff
@@ -1,25 +1,25 @@
 ---
-no_native_review: true
+no_native_review: false
```

Comment:
This change is not necessary. The tag will be removed on merge by a maintainer (if someone other than the author reviewed the article, which is the case here as it seems to me). I would recommend you to mention the removal of the tag in the PR's description for better visibility.

Reply:
Oh, I didn't know about that. I did that on all the articles I submitted and they had that tag. I will keep that in mind for future articles I submit.
Sorry about that.

Reply:
No worries! I wanted to help you get used to the wiki procedures more easily.